traitcuration/traits/datasources/zooma.py

```diff
@@ -0,0 +1,84 @@
+"""
+This module contains all the functionality that uses ZOOMA to retrieve mapping suggestions and create the suggested
+terms in the app's database
+"""
+import requests
+
+from ..models import Trait, MappingSuggestion, OntologyTerm, User
+from .ols import make_ols_query, get_ontology_id
+
+
+BASE_URL = "http://www.ebi.ac.uk/spot/zooma/v2/api"
+
+
+def get_zooma_suggestions():
+    """
+    Retrieves zooma mapping suggestions for all traits and creates terms and suggestions in the app's database for each
+    of them.
+    """
+    traits = Trait.objects.all()
+    for trait in traits:
+        results_list = get_suggestions_for_trait(trait)
+        for result in results_list:
+            term = create_local_term(result)
+            # If local term creation returns None, it means that this term already exists in the database
```

Comment:
Not sure if I get the logic here. What about the case when the term _is_ already in the database? Looks like `if term is not None` will be False and the mapping suggestion will not be created. But this shouldn't be the case. Consider the example:
* Originally "Trait A" had a ZOOMA suggestion to TERM:123456
* During next update iteration, we ingest "Trait A, some subtype", and ZOOMA also suggests to map it to TERM:123456. In this case, we should just create the mapping suggestion linking "Trait A, some subtype" to the _existing_ term.

Reply:
This can be most easily implemented by returning the existing term from the database (instead of None) in `create_local_term`